.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
`.vscode` folder should be under gitignore

Reply:
yes bhaia, my bad